components/GeoApi.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import { resolve } from "styled-jsx/css";
+import style from "../styles/Home.module.css";
+
+function GeoApi(props) {
+  const { weatherForecast, setWeatherForecast } = props;
+  const [locationName, setLocationName] = useState("Linn, Kansas");
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleChange = (e) => {
+    setSearchTerm(e.target.value);
+  };
+
+  async function fetchCoords() {
+    let long = 0;
+    let lat = 0;
+    let firstRes = await fetch(`http://api.positionstack.com/v1/forward?access_key=${process.env.NEXT_PUBLIC_GEOLOCATION_API_KEY}&query=${searchTerm}`);
```

Comment:
To help out unsuspecting future devs, it'd be nice to move the env to the top of the file and add some custom validation if the key isn't set like:

```js
const API_KEY = process.env.NEXT_PUBLIC_GEOLOCATION_API_KEY
// if API_KEY is empty throw error describing this key needs to be set
```

What do you think?

Reply:
Yeah, that make sense. I'll add it.